lib/MusicLibraryController.rb

```diff
@@ -0,0 +1,95 @@
+require 'pry'
+
+extend Concerns::Findable
+
+class MusicLibraryController
+
+  attr_accessor :path
+
+  def initialize(path = "./db/mp3s")
+    new_importer_obj = MusicImporter.new(path)
+    new_importer_obj.import
+  end
+
+
+  def call
+    input = ""
+    while input != "exit"
+      puts "Welcome to your music library!"
+      puts "To list all of your songs, enter 'list songs'."
+      puts "To list all of the artists in your library, enter 'list artists'."
+      puts "To list all of the genres in your library, enter 'list genres'."
+      puts "To list all of the songs by a particular artist, enter 'list artist'."
+      puts "To list all of the songs of a particular genre, enter 'list genre'."
+      puts "To play a song, enter 'play song'."
+      puts "To quit, type 'exit'."
+      puts "What would you like to do?"
+      input = gets.strip
+      case input
+      when "list songs"
+        list_songs
+      when "list artists"
+        list_artists
+      when "list genres"
+        list_genres
+      when "list artist"
+        list_songs_by_artist
+      when "list genre"
+        list_songs_by_genre
+      when "play song"
+        play_song
+      end
+    end
+  end
+
+  def list_songs
+    Song.all.sort{ |a, b| a.name <=> b.name }.each.with_index(1) do |s, i|
```

Comment:
Can this be accomplished with `sort_by(:name)`?

Reply:
Updated the `list_songs`, `list_artist`, `list_genres`, & `list_songs_by_artist`, `list_songs_by_genre`, & `play_song` to use the `sort_by(&:name)` instead of `sort{ |a, b| a.name <=> b.name }`.